core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -310,6 +310,11 @@ angular.module('oppia').directive('adminRolesTab', [
           refreshFormData();
           ctrl.resultRolesVisible = false;
           ctrl.contributionReviewersDataFetched = false;
+          $scope.$watch(
```

Comment:
Instead of adding an explicit watcher can we use `ng-change` in the `select` tag and mark `ctrl.contributionReviewersDataFetched = false;` whenever the selected value changes?

Reply:
Hi, I just switched to using ng-change!